server/service/apple_mdm_test.go

```diff
@@ -2110,7 +2127,7 @@ func TestUpdateMDMAppleSetup(t *testing.T) {
 
 func TestMDMAppleReconcileAppleProfiles(t *testing.T) {
 	ctx := context.Background()
-	mdmStorage := &mock.MDMAppleStore{}
+	mdmStorage := &mdmmock.MDMAppleStore{}
```

Comment:
Looks like we need an implementation of `mdmStorage.GetAllMDMConfigAssetsByNameFunc`?